### PR TITLE
acrn-config: assign PCPU0~1 to post vm by default

### PR DESCRIPTION
**Pull request by liuwei987**
- fix log macros for board defconfig
- add clearlinux UOS for launch config xmls
- enable gvtd for waag for nuc7/nuc6 board
- add 4 POST_STD_VM and 1 KATA_VM for industry config xmls
- assign PCPU0-1 to post vm by default
- parse KATA VM count number with vm_type 'KATA_VM'
- modify CONFIG_HV_RAM_START for NUC7i7DNB

Tracked-On: #4641
Signed-off-by: Wei Liu <weix.w.liu@intel.com>
Acked-by: Victor Sun <victor.sun@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by binbinwu1** [comment]
@terryzouhao Could you have a review?

**Comment by binbinwu1** [comment]
> Assign PCPU0~1 to post VM , PCPU2~3 to RT-VM for industrial xml config. also NUC support.

Hi @terryzouhao  github treat two '~' as delete notation : )

File: hypervisor/arch/x86/configs/nuc7i7dnb.config [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Generated by Kconfiglib (https://github.com/ulfalizer/Kconfiglib)
 CONFIG_BOARD="nuc7i7dnb"
 CONFIG_SERIAL_LEGACY=y
-CONFIG_HV_RAM_START=0x60000000
+CONFIG_HV_RAM_START=0x41000000
```
Comment (binbinwu1) [inline hypervisor/arch/x86/configs/nuc7i7dnb.config:4]: @liuwei987  Could you add a description why we modify HV RAM START address in this PR?

**Comment by jsun26intel** [comment]
@liuwei987, please use a separate patch to track.
When boot ACRN hypervisor from grub multiboot, HV will be loaded at CONFIG_HV_RAM_START since relocation is not supported in grub multiboot1. The CONFIG_HV_RAM_SIZE in industry scenario will take ~330MB(0x14000000), unfortunately the efi memmap on NUC7i7DNB is truncated at 0x6dba2000 although it is still usable from 0x6dba2000. So from grub point of view, it could not find a continuous memory from 0x6000000 to load industry scenario. Per efi memmap, there is a big memory area available from 0x40400000, so put CONFIG_HV_RAM_START to 0x41000000 is much safe for NUC7i7DNB.

**Comment by liuwei987** [comment]
OK，I will separate out 4 patch for this PR.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by wenlingz** [comment]
@binbinwu1  please help review again, this PR has updated, thanks.

**Pull request merged**